inhouse/db_util.py

```diff
@@ -18,25 +18,44 @@ def complete_transaction(self, cursor):
 
     async def get_names(self):
```

Comment:
One thing that I think will help us a lot (doesn't need to be this MR and I've already started down the path anyway), just adding it here for posterity, is to move all of our DB interactions into this handler class and wrap in the try/excepts with rollbacks.

Like I said I am already in the process of doing that but just going forward I think that will help a lot of our DB woes.

Reply:
I think we fix this when we do the error handling overhaul

Reply:
Largely fixed with https://github.com/csconner1998/InhouseBot/pull/28